otherlibs/dune-rpc/dune_rpc.mli

```diff
@@ -17,11 +17,7 @@
     that uses fibers *)
 
 module V1 : sig
-  module Sexp : sig
-    type t =
-      | Atom of string
-      | List of t list
-  end
+  open Dune_csexp (* TODO remove this *)
```

Comment:
@jeremiedimino I think we need to switch to the public csexp in dune-private-libs to get rid of this.

Reply:
Yes, that seems fine. In particular, given that `dune-rpc` is public, it seems better to depend on the public csexp rather than on the vendored version.